web/app/js/components/Top.jsx

```diff
@@ -156,14 +162,16 @@ class Top extends React.Component {
           resourcesByNs={this.state.resourcesByNs}
           authoritiesByNs={this.state.authoritiesByNs}
           tapRequestInProgress={this.state.tapRequestInProgress}
+          tapIsClosing={this.state.tapIsClosing}
```

Comment:
we'll need to set a default value for this state otherwise we get a console error on the Top page:

```
warning.js:33 Warning: Failed prop type: The prop `tapIsClosing` is marked as required in `TapQueryForm`, but its value is `undefined`.
    in TapQueryForm (created by AddUrlProps(TapQueryForm))
    in AddUrlProps(TapQueryForm) (at Top.jsx:158)
    in div (at Top.jsx:155)
    in Top (at AppContext.jsx:10)
    in Unknown (created by Route)
    in Route (at index.js:56)
    in Switch (at index.js:47)
    in div (at index.js:46)
    in div (created by Basic)
    in Basic (created by Adapter)
    in Adapter (at index.js:45)
    in div (created by BasicLayout)
    in BasicLayout (created by Adapter)
    in Adapter (at index.js:43)
    in div (created by BasicLayout)
    in BasicLayout (created by Adapter)
    in Adapter (at index.js:41)
    in RouterToUrlQuery (at index.js:40)
    in Router (created by BrowserRouter)
```